ui/job-view/pushes/PushList.jsx

```diff
@@ -314,12 +314,15 @@ export default class PushList extends React.Component {
 
   // Clear the selectedJob
   closeJob() {
+    const { pinnedJobs } = this.props;
+
     // TODO: Should block clearing the selected job if there are pinned jobs
     // But can't get the pinned jobs at this time.  When we're completely on React,
     // or at least have a shared parent between PushList and DetailsPanel, we can share
     // a PinBoardModel or Context so they both have access.
```

Comment:
Does this TODO comment need updating now, or is the "when we're completely on React" comment still apply?

Reply:
Yeah, I think I could update this comment.  The next PR creates a ``SelectedJob`` context, which will move and modify this function.  I'll update the comment in that PR, if that's cool.  Otherwise it may get lost in conflict resolution.  :)